tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDtdData.java

```diff
@@ -432,8 +443,17 @@ public void TestNewDtdData() {
             "row" // keyboard
             )));
 
+    static final Set<String> orderedKeyboardTestElements = Collections.unmodifiableSet(new HashSet<>(Arrays
+            .asList(    "repertoire", "event", "emit", "keystroke", "check", "backspace" )));
```

Comment:
Why is `repertoire` in this list? And I think `event` should be deleted?

Reply:
fixed